src/Microsoft.TestPlatform.Extensions.BlameDataCollector/PublicAPI/PublicAPI.Shipped.txt

```diff
@@ -31,7 +31,7 @@ Microsoft.TestPlatform.Extensions.BlameDataCollector.IBlameReaderWriter
 Microsoft.TestPlatform.Extensions.BlameDataCollector.IBlameReaderWriter.ReadTestSequence(string filePath) -> System.Collections.Generic.List<Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameTestObject>
 Microsoft.TestPlatform.Extensions.BlameDataCollector.IBlameReaderWriter.WriteTestSequence(System.Collections.Generic.List<System.Guid> testSequence, System.Collections.Generic.Dictionary<System.Guid, Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameTestObject> testObjectDictionary, string filePath) -> string
 Microsoft.TestPlatform.Extensions.BlameDataCollector.ICrashDumper
-Microsoft.TestPlatform.Extensions.BlameDataCollector.ICrashDumper.AttachToTargetProcess(int processId, string outputDirectory, Microsoft.TestPlatform.Extensions.BlameDataCollector.DumpTypeOption dumpType, bool collectAlways) -> void
+Microsoft.TestPlatform.Extensions.BlameDataCollector.ICrashDumper.AttachToTargetProcess(int processId, string outputDirectory, Microsoft.TestPlatform.Extensions.BlameDataCollector.DumpTypeOption dumpType, bool collectAlways, System.Action<string> logWarning) -> void
```

Comment:
Safe to do, I did it few times in the past as well.. This should not even be public api.